src/coverlet.core/TeamCityServiceMessageWriter.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Coverlet.Core;
+using System;
+
+namespace coverlet.core
+{
+    public class TeamCityServiceMessageWriter
```

Comment:
Nest this class in `TeamCityReporter` or at least move it to the file. Code specific to a reporter should be contained in the file/class

Reply:
It should also be private or internal

Reply:
Merged the file into the reporter